.github/workflows/copilot-build-frontend.yml

```diff
@@ -1,35 +0,0 @@
-name: copilot-build-frontend
```

Comment:
Why are we removing this workflow?

Reply:
It actually isn't / shouldn't be used.

Reply:
this workflow is and should be used. it is triggered by Github in PRs when any changes are made to the `webapp/` folder and is responsible for ensuring that the build/format is passing:
<img width="698" alt="image" src="https://github.com/microsoft/chat-copilot/assets/52973358/7805bfe9-cfc4-4ea0-aebb-2683a6e7bdd5">

Reply:
@dehoward You are totally right!  For some reason, when I removed it, I just saw it as a dup of the frontend deployment workflow and didn't think of the PR workflow.  Restoring this file.